zap/src/main/java/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -296,7 +296,7 @@ protected void sendAndReceive(
 
         if (this.getDelayInMs() > 0) {
             try {
-                Thread.sleep(this.getDelayInMs());
```

Comment:
Changing things like this is fine but would be much better in another 'clean up' PR - it detracts from the important changes.

Reply:
true, I took the boy-scout approach on the files I changed a lot